telegram/ext/filters.py

```diff
@@ -56,13 +58,21 @@ class variable.
 
     Attributes:
         name (:obj:`str`): Name for this filter. Defaults to the type of filter.
-
+        update_filter (:obj:`bool`): whether this filter should work on update. If ``False`` it
+            will run the filter on :attr:`update.effective_message``. Default is ``False``.
+        regex_filter (:obj:`bool`): Whether this filter is a regexfilter. Will return a list with
+            the match object instead of a bool.
     """
 
     name = None
+    update_filter = False
+    regex_filter = False
```

Comment:
Maybe this should have a more generic name? As filters might need to return other things than match objects and bools in the future?

Reply:
True, but for now it describes what it is. If it needs to change we can do so, it;s all internal anyway

Reply:
I'm not sure I agree that it's internal. The BaseFilter is public and we actively encourage users to use it if they need to make their own filters.